frontend/src/components/_common/Calendar/Calendar.styles.ts

```diff
@@ -60,29 +85,95 @@ export const s_daySlotButton = css`
   min-width: 3.6rem;
   height: 3.6rem;
 
-  background-color: transparent;
-  border: none;
+  /* 모바일에서 계란처럼 보이는 이유가 버튼에 기본 padding이 있는 문제라서, 0으로 임시 해결하긴 했습니다. 글로벌 스타일 버튼을 어떻게 만들어가야할지는 논의가 필요할 것 같아요(@해리) */
+  padding: 0;
 `;
 
-// TODO : 공휴일 색 변경 논의 필요(@해리)
-// TODO : s_todayDaySlot 추가 예정(@해리)
-export const s_daySlot = (isHoliday: boolean) => css`
+export const s_daySlotButton = css`
   cursor: pointer;
-  font-size: 1.5rem;
-  color: ${isHoliday ? theme.colors.holiday : theme.colors.black};
+  background-color: transparent;
+  border: none;
+  ${theme.typography.bodyLight}
+
+  &:disabled {
+    cursor: default;
+  }
 `;
 
-export const s_selectedDaySlot = (isSelected: boolean) => css`
+export const s_baseDaySlotText = css`
   display: flex;
   align-items: center;
   justify-content: center;
 
   width: 3.6rem;
   height: 3.6rem;
+`;
 
-  ${isSelected &&
-  css`
-    background-color: #fcc;
+export const s_daySlotText = (
+  isHoliday: boolean,
+  isSelected: boolean,
+  isToday: boolean,
+  isSaturday: boolean,
+  isPrevDay: boolean,
+) => {
+  /* 덕지덕지 if문인데 어쩔 수 없다고 생각하기는 했습니다. 가독성을 위해서 switch문을 사용하는 것도 고려해보면 좋을 것 같은데, 코멘트로 의견 부탁드려요(@해리) */
+  /* if문 위에서부터 아래로, 스타일이 적용되어야 하는 우선순위입니다. 선택된 날짜의 스타일이 가장 우선적으로 고려되어야 하고, 그 다음은 지난날짜,,,순 입니다. 그래서 early return 패턴을 활용했어요(@해리) */
+  if (isSelected) {
+    return css`
+      ${DAY_SLOT_TEXT_STYLES.selected}
+    `;
+  }
+
+  if (isPrevDay) {
+    return css`
+      ${DAY_SLOT_TEXT_STYLES.prevDay}
+    `;
+  }
+
+  if (isHoliday) {
+    return css`
+      ${DAY_SLOT_TEXT_STYLES.holiday}
+    `;
+  }
+
+  if (isToday) {
+    return css`
+      ${DAY_SLOT_TEXT_STYLES.today}
+    `;
+  }
+
+  if (isSaturday) {
+    return css`
+      ${DAY_SLOT_TEXT_STYLES.saturday}
+    `;
+  }
+
+  return css`
+    ${DAY_SLOT_TEXT_STYLES.default}
+  `;
+};
```

Comment:
[C]

아직 더 좋은 로직은 모르겠습니다 ㅠㅠ 하지만, 이렇게 짧게 줄여볼 수도 있겠네요!

```suggestion
export const s_daySlotText = (
  isHoliday: boolean,
  isSelected: boolean,
  isToday: boolean,
  isSaturday: boolean,
  isPrevDay: boolean,
) => {
  /* 덕지덕지 if문인데 어쩔 수 없다고 생각하기는 했습니다. 가독성을 위해서 switch문을 사용하는 것도 고려해보면 좋을 것 같은데, 코멘트로 의견 부탁드려요(@해리) */
  /* if문 위에서부터 아래로, 스타일이 적용되어야 하는 우선순위입니다. 선택된 날짜의 스타일이 가장 우선적으로 고려되어야 하고, 그 다음은 지난날짜,,,순 입니다. 그래서 early return 패턴을 활용했어요(@해리) */
  if (isSelected) return DAY_SLOT_TEXT_STYLES.selected;
  if (isPrevDay) return DAY_SLOT_TEXT_STYLES.prevDay;
  if (isHoliday) return DAY_SLOT_TEXT_STYLES.holiday;
  if (isToday) return DAY_SLOT_TEXT_STYLES.today;
  if (isSaturday) return DAY_SLOT_TEXT_STYLES.saturday;

  return DAY_SLOT_TEXT_STYLES.default;
};
```

Reply:
이 방법이 더 좋겠군요~ 덕지덕지 if문이 어쩔 수 없는 것이라면 저렇게라도 가독성을 챙겨야겠습니다 ㅎㅎ

Reply:
**[C]**

분기 처리가 많은 상황이라면 switch 문을 적극적으로 도입해봐도 좋을 것 같네요!
저는 개인적으로 4가지 이상의 케이스가 생기면 switch 문 사용하는 것 같아요

Reply:
# TODO 

switch문 도입 컨벤션 정하기